web/yarn.lock

```diff
@@ -8581,9 +6772,9 @@ yargs-parser@^13.1.2:
     decamelize "^1.2.0"
 
 yargs-parser@^20.2.2, yargs-parser@^20.2.3:
-  version "20.2.7"
-  resolved "https://registry.yarnpkg.com/yargs-parser/-/yargs-parser-20.2.7.tgz#61df85c113edfb5a7a4e36eb8aa60ef423cbc90a"
-  integrity sha512-FiNkvbeHzB/syOjIUxFDCnhSfzAL8R5vs40MgLFBorXACCOAEaWu0gRZl14vG8MR9AOJIZbmkjhusqBYZ3HTHw==
+  version "20.2.9"
+  resolved "https://registry.yarnpkg.com/yargs-parser/-/yargs-parser-20.2.9.tgz#2eb7dc3b0289718fc295f362753845c41a0c94ee"
+  integrity sha512-y11nGElTIV+CT3Zv9t7VKl+Q3hTQoT9a1Qzezhhl6Rp21gJ/IVTW7Z3y9EWXhuUBC2Shnf+DX0antecpAwSP8w==
```

Comment:
Kind of incredible that the lockfile's shrunk by about 2K lines from those package changes. Looks like it comes mostly from removing img-optimize-loader? :exploding_head:

Reply:
It carries multiple dependencies - including optionalDependencies and their dependencies. You get the... picture 😄 
```
  dependencies:
    imagemin "^7.0.1"
    loader-utils "^2.0.0"
    mime-types "^2.1.27"
    schema-utils "^2.7.0"
  optionalDependencies:
    imagemin-gifsicle "^6.0.1"
    imagemin-mozjpeg "^8.0.0"
    imagemin-optipng "^7.1.0"
    imagemin-pngquant "^8.0.0"
    imagemin-svgo "^7.0.0"
    imagemin-webp "^5.1.0"
```

Also, the newest lockfile is built against LTS Node v16.